gems/smithy/lib/smithy/generators/base.rb

```diff
@@ -10,8 +10,9 @@ class Base
       # @param [Plan] plan The plan to generate.
       def initialize(plan)
         # Necessary for Thor::Base and Thor::Actions
-        self.options = { force: true }
-        self.destination_root = plan.options[:destination_root]
+        self.options = { force: true, quiet: plan.quiet }
+        self.destination_root = plan.destination_root
+        shell.base = self
```

Comment:
Whats this doing?

Reply:
Typically when you inherit from Thor, it does some setup, but this is the minimal setup for including thor actions and base. The thor shell is what controls output. The shell's base (the class) is nil, so quiet? method ends up returning false always. Now that base is not nil, quiet option is respected.

Reply:
Ah I see!  I think that was the key piece I was missing when I tried to get quiet to work.